src/analysis/processing/qgsalgorithmrasterfrequencybycomparisonoperator.cpp

```diff
@@ -246,14 +246,15 @@ QString QgsRasterFrequencyByEqualOperatorAlgorithm::shortHelpString() const
 {
   return QObject::tr( "The Equal to frequency algorithm evaluates on a cell-by-cell basis the frequency "
                       "(number of times) the values of an input stack of rasters are equal "
-                      "to the value of a value raster. \n "
+                      "to the value of a reference raster layer. \n "
                       "If multiband rasters are used in the data raster stack, the algorithm will always "
-                      "perform the analysis on the first band of the rasters - use GDAL to use other bands in the analysis. "
-                      "The input value layer serves as reference layer for the sample layers. "
+                      "perform the analysis on the first band of the rasters - use GDAL to use other bands in the analysis.\n "
```

Comment:
Any idea which GDAL commands/algs can be used here? We can point to them in the online documentation (https://github.com/qgis/QGIS-Documentation/pull/6872)

Reply:
When I wrote the algorithm I was thinking about `gdal:translate` algorithm.

Reply:
Probably the "rearrange bands" algorithm is the correct one here